easybuild/easyconfigs/n/NWChem/NWChem-6.6.revision27746-parallelbuild.patch

```diff
@@ -0,0 +1,80 @@
+diff -ru nwchem-6.6.orig/src/config/makefile.h nwchem-6.6/src/config/makefile.h
```

Comment:
@akesandgren can you include a brief description of this patch at the top, explaining what it does, where it comes from and/or who the author is?